app/src/main/java/de/mathfactory/mooltifill/SettingsActivity.kt

```diff
@@ -231,14 +236,35 @@ class SettingsActivity : AppCompatActivity() {
         }
 
         private fun rescan() = CoroutineScope(Dispatchers.Main).launch {
+            val activity = activity ?: return@launch
+
+            if (!PermissionUtils.hasBluetoothPermission(activity)) {
+                val bluetoothConnectPermission = activity.registerForActivityResult(ActivityResultContracts.RequestPermission()) {
+                    if (it) {
+                        scanBluetoothDevices()
+                    }
+                }
+
+                if (shouldShowRequestPermissionRationale(Manifest.permission.POST_NOTIFICATIONS)) {
+                    Snackbar.make(activity.window.decorView.rootView,
+                        getString(R.string.enable_bluetooth_permission_message), Snackbar.LENGTH_SHORT).show()
```

Comment:
shouldShowRequestPermissionRationale is called with Manifest.permission.POST_NOTIFICATIONS (which is about notification permission), but the string shown is R.string.enable_bluetooth_permission_message (which is about enabling bluetooth)?

Reply:
@mathfactory Oops, my bad. I have updated it with the correct permission string now.
Thanks for pointing out.